.github/workflows/pull_request.yml

```diff
@@ -13,11 +13,21 @@ on:
   workflow_dispatch:
 
 concurrency:
-  group: build-${{ github.ref }}
+  group: Pull-Request-${{ github.ref }}
   cancel-in-progress: true
 
 jobs:
-  buildandtest:
-    name: Build and Test
-    uses: ./.github/workflows/build-and-test.yml
-    secrets: inherit
+  reuse_action:
+    name: REUSE Compliance Check
+    uses: StanfordSpezi/.github/.github/workflows/reuse.yml@v2
+  swiftlint:
+    name: SwiftLint
+    uses: StanfordSpezi/.github/.github/workflows/swiftlint.yml@v2
+  markdown_link_check:
+    name: Markdown Link Check
+    uses: StanfordBDHG/.github/.github/workflows/markdown-link-check.yml@v2
+  breaking_changes:
+    name: Diagnose Breaking Changes
+    uses: StanfordSpezi/.github/.github/workflows/breaking-changes.yml@v2
+    with:
+      runsonlabels: '["macOS", "self-hosted"]'
```

Comment:
Great here, will probably have to remove this in the template repo when we contribute this back.